decentraland/helpers/storage.js

```diff
@@ -118,6 +129,15 @@ const _storeData = async (key, value, securely) => {
   }
 };
 
+const _clearData = async key => {
+  try {
+    await AsyncStorage.removeItem(key);
+    await SecureStore.deleteItemAsync(key);
```

Comment:
Do we only want to do this if it failed to remove it from AsyncStorage? Or regardless. I guess the current approach is more defensive which is probably good.

Reply:
Yes, I've tried to write a more defensive.
Your question makes me think if the behavior when failing, the docs aren't clear about that so I've changed this function to make sure that both functions will be executed.

Reply:
Oh, that was already merged. I'll create a new PR.

Reply:
> Yes, I've tried to write a more defensive.
> Your question makes me think if the behavior when failing, the docs aren't clear about that so I've changed this function to make sure that both functions will be executed.

Good thinking. Sounds good. Resolving this.